### ISSUE TITLE
Request Entity Too Large

### ISSUE THREAD

**Issue by gonzalo-roberto-diaz** [aaronshaf/dynamodb-admin#40]
I work with large table entries that contain songs.
Each song has a large "lyrics" entry, and the a JSon array of objects that contain each individual word, plus some extra information (what part of speech is, what gender, number, etc).

Often I have to edit one of those words, say, in the middle of the array, and then hit the "Save" button, but get a `Request Entity too large` error.

I know it is not the length of the request per se, because I found a workaround: 
* removing the large song lyric entry
* editing my word entry in the array
* re-ingest the large lyric entry

This works, and results in a song table entry containing what I wanted, and is not "too large".

Here is an example stack

```
PayloadTooLargeError: request entity too large
    at readStream (/usr/local/lib/node_modules/dynamodb-admin/node_modules/raw-body/index.js:155:17)
    at getRawBody (/usr/local/lib/node_modules/dynamodb-admin/node_modules/raw-body/index.js:108:12)
    at read (/usr/local/lib/node_modules/dynamodb-admin/node_modules/body-parser/lib/read.js:77:3)
    at jsonParser (/usr/local/lib/node_modules/dynamodb-admin/node_modules/body-parser/lib/types/json.js:135:5)
    at Layer.handle [as handle_request] (/usr/local/lib/node_modules/dynamodb-admin/node_modules/express/lib/router/layer.js:95:5)
    at next (/usr/local/lib/node_modules/dynamodb-admin/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/usr/local/lib/node_modules/dynamodb-admin/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/usr/local/lib/node_modules/dynamodb-admin/node_modules/express/lib/router/layer.js:95:5)
    at /usr/local/lib/node_modules/dynamodb-admin/node_modules/express/lib/router/index.js:281:22
    at param (/usr/local/lib/node_modules/dynamodb-admin/node_modules/express/lib/router/index.js:354:14)
```

Is this an error, or I am misusing the tool?
I am next to null with nodeJS, but can help trying to find a solution, if someone gives me an idea where to look.

**Comment by rchl** [comment]
Fixed with #41

**Comment by gonzalo-roberto-diaz** [comment]
Excellent, thank you!
